### ISSUE TITLE
Extend kafka producer configuration

### ISSUE THREAD

**Issue by AntonDambrouski** [hayrullahcansu/nlog-kafka-target#7]
The kafka appender works great, but it doesn't allow to write logs to kafka that requires the certificate.
So, to make it work I need to be able to specify path to **ssl certificate location** and set **security protocol** to Ssl.
These properties might be configured during producer building.
E.g. 
```
var conf = new ProducerConfig
            {
                BootstrapServers = brokers,
                SslCertificateLocation = configs?.SslCertificateLocation,
                SecurityProtocol = configs?.SecurityProtocol,
            };
```

**Comment by hayrullahcansu** [comment]
Hi @AntonDambrouski i see your PR, Today i will review it and get back to you. Thanks for collaborating.

**Comment by AntonDambrouski** [comment]
@hayrullahcansu, Thank you!

**Comment by hayrullahcansu** [comment]
@AntonDambrouski a bug is fixed with new version (2.1.1). [ref-PR](https://github.com/hayrullahcansu/nlog-kafka-target/pull/9)
